ArduCopter/mode_drift.cpp

```diff
@@ -54,7 +54,7 @@ void ModeDrift::run()
 
     // gain scheduling for yaw
     float pitch_vel2 = MIN(fabsf(pitch_vel), 2000);
-    float target_yaw_rate = ((float)target_roll/1.0f) * (1.0f - (pitch_vel2 / 5000.0f)) * g.acro_yaw_p;
+    float target_yaw_rate = target_roll * (1.0f - (pitch_vel2 / 5000.0f)) * g2.acro_y_rate / 45.0;
```

Comment:
Why the `/ 45.0` ?

Reply:
I believe this stems from the output to AP_Motors being an angle which is +/- 4500 cdeg.  So now that we are going to a rate param instead of a gain, we need to convert it back to a gain for equivilent behaviour here.  Looks correct to me.

Reply:
Please add that to a comment in the code.

Reply:
mode drift is not my code so I can't comment why it is that way. If I had my way i would just delete the mode.

Reply:
I'm not really a fan either, I suspect we could persuade @rmackay9 ....

Reply:
I'm certainly a fan of simplifying Copter to remove features that almost nobody uses.  Undoubtedly someone will turn up and say they use the mode (it came up once during 4.1 beta testing) but we could argue for the sake of supportability and progress that it should go.  The first step would be to make it a compiled-off-by-default option and then a version later we remove it assuming there haven't been too many complaints.